api/documentation/docs/understanding-our-api/resources/collective-offers.md

```diff
@@ -1,7 +1,84 @@
+:::
+
+A bookable offer is linked to (at least) one collective booking. Most of the time, there is only be one booking for one offer. However, if one is cancelled, a new one can be created. The offer now has two bookings: an ongoing one and a cancelled one.
+
+:::info
+There can only be one active booking per bookable offer.
+:::
+
+### Status
+
+A bookable offer can have five states:
+
+* `PENDING`
+* `CONFIRMED`
+* `USED`
+* `CANCELLED`
+* `REIMBURSED`
+
+`PENDING` means it has been proposed to the teacher. Once both sides are ok with the price, details, etc. it becomes `CONFIRMED`. It becomes `USED` when the event is passed and `REIMBURSED` after a short period of time (check the official documentation to get more detailed financial information). If the offer is cancelled... it becomes `CANCELLED`, obviously.
+
+### Event location
+
+There are two location types: at school or elsewhere. For the latter, there are two ways to specify the location: either using a venue's id (from our database, usually one of your venue), or using a free text field.
+
+The location must be specified under the `offerVenue` object which contains three fields:
+
+* `addressType` (`OFFERER_VENUE`, `SCHOOL` or `OTHER`);
+* `venueId` (null when type is `SCHOOL` or `OTHER`);
+* `otherAddress` (not null only if `addressType` equals `OTHER`).
+
+## Creation rules
+
+### Offer templates
+
+One cannot create a collective offer using the public API (for now).
+
+### Bookable offer
+
+A bookable offer can be created using the `post_collective_offer_public` endpoint [described here](/rest-api/#tag/)
+
+There are a two requirements to meet in order to create a collective offer:
+
+* your offerer must be referenced by Adage;
+* the offer's subcategory is valid and eligible for a collective offer.
+
+An offer created using the public API cannot be edited using the PRO web portal.
+
+## Update rules
+
+### Offer templates
+
+They cannot be updated using the API (for now).
+
+### Bookable offers
+
+An offer can only be updated if it is in the `PENDING` state.
+
+## Collective bookings
+
+A collective booking is linked to a collective offer and works for a group of students.
+
+:::info
+Meaning: if a school project is booked for a 30 students class, there will be only one booking, not thirty.
+:::
+
+Also, the validation process is not the same as individual offers since these offers are meant to be a part of a larger school/teaching project. The main steps are: build a bookable offer with a teacher and pass through the hierarchy validation. For more details, please check the pass Culture's global documentation or contact the EAC support as these rules and timelines are outside of this (technical) documentation scope.
+
+The API will return the list of an offer's bookings and the possibility to cancel a booking. A collective booking cannot be cancelled anytime, once again, please check the global documentation or ask the support for more details.
```

Comment:
`The API will return the list of an offer's bookings and the possibility to cancel a booking.` Je ne suis pas sûr de bien comprendre cette ligne. Est-ce qu'il ne manque pas un mot ici ` and the possibility to cancel a booking` (genre "give").